Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -226,8 +186,9 @@ function FetchDiscussionComments
     
     foreach ($discussionThread in $discussionThreads)
     {
-        $commentsFromThread = [PsWorkarounds.Helper]::GetComments($script:discussionClient, $discussionThread.DiscussionId)
-        
+        $request = InvokeByReflection  $script:discussionClient "GetCommentsAsync" @([Int]) @($discussionThread.DiscussionId)
```

Comment:
No, this is there is only one GetComments method and it requires a discussion id, which is related to threads